10_deploy_rook.sh

```diff
@@ -64,3 +79,11 @@ spec:
     activeCount: 1
     activeStandby: true
 EOF
+
+cd $ROOKPATH/cluster/examples/kubernetes/ceph/monitoring
+sed 's/namespace: rook-ceph/namespace: openshift-storage/' service-monitor.yaml > service-monitor-modified.yaml
+sed -i 's/- rook-ceph/- openshift-storage/' service-monitor-modified.yaml
+oc create -f service-monitor-modified.yaml
+
+cd $MIXINPATH/extras/manifests
+oc create -f prometheus-rules.yaml
```

Comment:
The rule file name is going to be same for dev script as well?

Reply:
Whatever we decide. Currently, there are no other files that I can apply.